pathology/tumor_detection/ignite/camelyon_train_evaluate.py

```diff
@@ -269,7 +269,7 @@ def main():
     parser.add_argument("--no-novograd", action="store_false", dest="novograd", help="deactivate novograd optimizer")
     parser.add_argument("--no-pretrain", action="store_false", dest="pretrain", help="deactivate Imagenet weights")
 
-    parser.add_argument("--cpu", type=int, default=8, dest="num_workers", help="number of workers")
+    parser.add_argument("--cpu", type=int, default=1, dest="num_workers", help="number of workers")
```

Comment:
May I know what is the reason to reduce it to 1

Reply:
Hi @drbeh, when num_workers is large, it may occasionally throw this error.
```
Traceback (most recent call last):
  File "/usr/lib/python3.8/multiprocessing/queues.py", line 245, in _feed
    send_bytes(obj)
  File "/usr/lib/python3.8/multiprocessing/connection.py", line 200, in send_bytes
    self._send_bytes(m[offset:offset + size])
  File "/usr/lib/python3.8/multiprocessing/connection.py", line 411, in _send_bytes
    self._send(header + buf)
  File "/usr/lib/python3.8/multiprocessing/connection.py", line 368, in _send
    n = write(self._handle, buf)
BrokenPipeError: [Errno 32] Broken pipe
```

Reply:
So maybe set it to 2, since we want to make sure that this tutorial would work with `num_workers > 1`. What do you think?

Reply:
I am OK with 1 or 2.

Thanks.

Reply:
ok, will update it.